### PR TITLE
Update small things on website

### PR DESCRIPTION
**Pull request by f97gujo**
Some spelling, some grammar, some syntax.

I made some mistakes along the way, but hopefully got it right at the end.

### CODE REVIEW

File: docs_source_files/content/getting_started_with_webdriver/locating_elements.en.md [hunk 1]
```diff
@@ -122,7 +122,7 @@ to have to retrieve each of the items individually, a superior
 technique for retrieving cheese is to make use of the pluralized
 version `findElements(By)`. This method returns a collection of web
 elements. If only one element is found, it will still return a
-collection (of one element). If no elements match the locator, an
+collection (of one element). If no element match the locator, an
```
Comment (diemol) [inline docs_source_files/content/getting_started_with_webdriver/locating_elements.en.md]:
```suggestion
collection (of one element). If no element matches the locator, an
```

File: docs_source_files/content/guidelines_and_recommendations/consider_using_a_fluent_api.en.md [hunk 2]
```diff
@@ -3,8 +3,8 @@ title: "Consider using a fluent API"
 weight: 8
 ---
 
-Martin Fowler coined the term "Fluent API". Selenium already
-implements something like this in their `FluentWait` class which is
+Martin Fowler coined the term ["Fluent API"](//www.martinfowler.com/bliki/FluentInterface.html). Selenium already
```
Comment (diemol) [inline docs_source_files/content/guidelines_and_recommendations/consider_using_a_fluent_api.en.md:6]: Great! Could you please add the same link to the other translations? To keep them uniform.

File: docs_source_files/content/guidelines_and_recommendations/improved_reporting.en.md [hunk 3]
```diff
@@ -10,4 +10,9 @@ reports that can generate xUnit or HTML formatted reports.  xUnit
 reports are popular for importing results to a Continuous Integration
 (CI) server like Jenkins, Travis, Bamboo, etc.  Here are some links
 for more information regarding report outputs for several languages.
-<!-- TODO: Add links.-->
\ No newline at end of file
+<!-- TODO: Add links.-->
+[NUnit 3 Console Runner](//github.com/nunit/docs/wiki/Console-Runner)
```
Comment (diemol) [inline docs_source_files/content/guidelines_and_recommendations/improved_reporting.en.md:14]: Great! Could you please add the same link to the other translations? To keep them uniform.

File: docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.en.md [hunk 4]
```diff
@@ -23,7 +23,7 @@ e = driver.find_element_by_id("testElement")
   {{< / code-panel >}}
 {{< / code-tab >}}
 
-You can simulate mouse clicking on e if it's visible and has a height and width
+You can simulate mouse clicking on e if itis visible and has a height and width
```
Comment (diemol) [inline docs_source_files/content/support_packages/mouse_and_keyboard_actions_in_detail.en.md]:
```suggestion
You can simulate mouse clicking on e if it is visible and has a height and width
```

File: docs_source_files/content/webdriver/browser_manipulation.en.md [hunk 5]
```diff
@@ -475,7 +478,7 @@ try {
 {{< / code-tab >}}
 
 Python's WebDriver now supports the python context manager,
-which when using the with keyword can automatically quit the driver at
+which when using the "with" keyword can automatically quit the driver at
```
Comment (diemol) [inline docs_source_files/content/webdriver/browser_manipulation.en.md]:
```suggestion
which when using the `with` keyword can automatically quit the driver at
```

File: docs_source_files/content/worst_practices/performance_testing.en.md [hunk 6]
```diff
@@ -38,11 +38,11 @@ you will need to be able to analyse overall performance
...
 in order to know what to improve.
 There are performance testing tools available
 that can do this job already,
-and which provide reporting and analysis
-which can even make improvement suggestions.
+that provide reporting and analysis,
+and can even make improvement suggestions.
 
-Example (open source) packages to use are: JMeter ?
+Example (open source) packages to use are: [JMeter](//jmeter.apache.org/)
```
Comment (diemol) [inline docs_source_files/content/worst_practices/performance_testing.en.md:48]: Great! Could you please add the same link to the other translations? To keep them uniform.

**Review by diemol — CHANGES_REQUESTED** [review]
Thanks @f97gujo!
Just a few comments.

File: docs_source_files/content/guidelines_and_recommendations/improved_reporting.en.md (hunk 3 above)
Comment (f97gujo) [reply docs_source_files/content/guidelines_and_recommendations/improved_reporting.en.md:14]: Updated.

File: docs_source_files/content/guidelines_and_recommendations/consider_using_a_fluent_api.en.md (hunk 2 above)
Comment (f97gujo) [reply docs_source_files/content/guidelines_and_recommendations/consider_using_a_fluent_api.en.md:6]: Updated.

File: docs_source_files/content/worst_practices/performance_testing.en.md (hunk 6 above)
Comment (f97gujo) [reply docs_source_files/content/worst_practices/performance_testing.en.md:48]: Updated.

**Review by diemol — APPROVED** [review]
Thank you @f97gujo!

**Pull request merged**